public/app/features/alerting/unified/components/receivers/form/ChannelSubForm.tsx

```diff
@@ -53,6 +53,7 @@ export function ChannelSubForm<R extends ChannelValues>({
 
   const { control, watch, register, trigger, formState, setValue } = useFormContext();
   const selectedType = watch(fieldName('type')) ?? defaultValues.type; // nope, setting "default" does not work at all.
+  const parse_mode = watch(fieldName('settings.parse_mode')) ?? defaultValues.settings?.parse_mode;
```

Comment:
The defaultValue of `parse_mode` is an object, isn't it? So this should be `parse_mode.value`?

Reply:
actually, when creating new contact point for telegram, in `defaultValues.settings.parse_mode` we have `undefined` (I've jsut checked). So I will remove the default value and change it only with  `const parse_mode = watch(fieldName('settings.parse_mode'))`